proxies.json

```diff
@@ -14,5 +14,14 @@
     "description": "Chris Ferebee",
     "detail": "./proxies/ferebee/detail.md",
     "networks": ["HNT", "MOBILE", "IOT"]
+  },
+  {
+    "wallet": "84Q9QQ1DeqsbeBMiaKuh2Vk1wrfKirkZXBrTKyLL2apU",
+    "name": "Jay M.",
+    "image": "./proxies/jaym2518/image.png",
+    "description": "Jay M.",
+    "detail": "./proxies/jaym2518/detail.md",
+    "networks": ["HNT", "MOBILE", "IOT"]
```

Comment:
just confirming you want to be assigned in all three networks?

Reply:
Good question. I removed IOT (I think I did, anyway..)

Reply:
you did it :)